code/modules/admin/sql_notes.dm

```diff
@@ -239,7 +239,7 @@
 			output += "<a href='?_src_=holder;shownoteckey=[index_ckey]'>[index_ckey]</a><br>"
 			CHECK_TICK
 		qdel(query_list_notes)
-		message_admins("The note search started by [usr.ckey] has complete. CPU should return to normal.")
```

Comment:
`Action Item A “is completed” is wrong, although “is being completed” or “is going to be completed” are proper verb forms.`

This was correct before.

Reply:
But it's not `is complete` currently, it is `has complete`. **Has** is used with a past participle to form the perfect, pluperfect, and future perfect tenses. The verb **complete** that means **to finish** is combined with the past tense form **completed**.
So this change is correct.